src/cli/serve/integration_tests/__snapshots__/invalid_config.test.js.snap

```diff
@@ -4,12 +4,15 @@ exports[`cli invalid config support exits with statusCode 64 and logs a single l
 Array [
   Object {
     "@timestamp": "## @timestamp ##",
+    "error": "## Error with stack trace ##",
+    "level": "fatal",
     "message": "\\"unknown.key\\", \\"other.unknown.key\\", \\"other.third\\", \\"some.flat.key\\", and \\"some.array\\" settings were not applied. Check for spelling errors and ensure that expected plugins are installed.",
     "pid": "## PID ##",
     "tags": Array [
       "fatal",
+      "root",
     ],
-    "type": "log",
+    "type": "error",
```

Comment:
>  I assume it has the same ultimate result?

Nope, the result is different, see screenshot below, top - Kibana 6.3, bottom - this PR:

![screenshot from 2018-09-06 08-26-07](https://user-images.githubusercontent.com/1713708/45136830-99596a80-b1ae-11e8-8a9d-383ea1a7963d.png)

This happens because we log entire error object [here](https://github.com/elastic/kibana/pull/22190/files#diff-8db9deb0b202d7fbbfbba2672f094593R73) and pass it to the Hapi/even-better [here](https://github.com/elastic/kibana/blob/1f7fdbfe7ca4dd5c07e290cb534184a97b4d44e3/src/core/server/legacy_compat/logging/legacy_logging_server.ts#L73) so log record gets `error` type and outputs stack trace I like the behavior in this PR, but maybe I'm too biased.

Do you see any problem with that change?

Reply:
Nah, the server doesn’t start anyway, so it doubt it will cause problems unless people are matching on whole log lines in some script that checks Kibana somehow... I’m not concerned